bin/minifollowups/pycbc_foreground_minifollowup

```diff
@@ -56,6 +57,13 @@ parser.add_argument('--analysis-category', type=str, required=False,
                     help='Designates whether to look at foreground triggers '
                          'background triggers (including "little dogs") '
                          'or background triggers (with "little dogs" removed)')
+parser.add_argument('--sort-variable', default='stat',
```

Comment:
Can you make the default the false-alarm rate? This was always the intended behaviour, stat was just a proxy for this.